cypress/features/regression/portrait.feature

```diff
@@ -8,4 +8,14 @@ Feature: Portrait default component
 		Examples:
 			| alt                          | nameOfObject        |
 			| mp150ú¿¡üßä                  | altOtherLanguage    |
-			| !@#$%^*()_+-=~[];:.,?{}&"'<> | altSpecialCharacter |
\ No newline at end of file
+			| !@#$%^*()_+-=~[];:.,?{}&"'<> | altSpecialCharacter |
+
+	@positive
+	Scenario Outline: Render specified icon type <iconType>
+		When I open icon "Portrait Test" component in noIFrame with "portrait" json from "commonComponents" using "<nameOfObject>" object name
+		Then <iconType> icon component should be rendered
+		Examples:
+			| iconType   | nameOfObject  |
+			| individual | iconTypeNone  |
```

Comment:
```suggestion
			| custom | iconTypeNone  |
```

Reply:
I do not understand this change request. The test is not setting a custom icon, but verifying that by default the "individual" icon is rendered if no other values are given.